.github/ISSUE_TEMPLATE/Emulation_bug_report.yaml

```diff
@@ -0,0 +1,76 @@
+# Docs - https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-githubs-form-schema
+name: Bug Report
+description: Report a issue with the emulation (ie. graphical issue, crashes, unintended behavior with doesn't occur on hardware).
+title: "[BUG]: "
+labels: [Bug]
+body:
+  - type: markdown
+    id: md_readme
+    attributes:
+      value: |
+        ## Important: Read First
+        
+        This is an issue tracker for tracking bugs and feature requests, and as such, please refrain from using it to submit support requests.
+        If you need any support links are provided in the readme.md
+        
+         ### Please Avoid Issues Pertaining to the Following:
+         -TO DO
+  - type: textarea
+    id: current_behavior
+    attributes:
+      label: Current Behavior
+      description: "What the bug is, in a breif decription"
+    validations:
+      required: true
+  - type: textarea
+    id: expected_behavior
+    attributes:
+      label: Expected Behavior
+      description: "What did you expect to happen?"
+    validations:
+      required: true
+
+  - type: textarea
+    id: steps_to_reproduce
+    attributes:
+      label: Steps to Reproduce
+      description: "How to reprodue the issue"
+    validations:
+      required: true
+  - type: textarea
+    id: sys_info
+    attributes:
+      label: System Info
+      description: "Place all system Info in the box below"
+      placeholder: |
+       | Info                | Sys Info        | 
+       |---------------------|-----------------|
+       | OS                  | Windows 10 |
+       | CPU                 | AMD Ryzen 5 5900X Six-Core Processor |
+       | GPU                 | NVIDIA GeForce RTX 3090 Ti      |
+       | Drivers Verison     | 4.0.0 NVIDIA 512.95      |
+       | Cemu Verison        | 2.0      | 
+      value: |
+       | Info                | Sys Info        | 
+       |---------------------|-----------------|
+       | OS                  | |
+       | CPU                 | |
+       | GPU                 | |
+       | Drivers Verison     | |
+       | Cemu Verison        | | 
```

Comment:
Markdown table is prone to editing errors by users. Would be better as separate fields, or just plain lines (no table).

Also 4 typos of `Verison`